src/components/ebay-notice/README.md

```diff
@@ -0,0 +1,22 @@
+# ebay-notice
+
+## `ebay-notice` tag
+The `<ebay-notice>` is a tag used to create a custom-designed notice element. The notice can be single or multi-line but each line should be wrapped inside a `<p>` tag.
+
+### `ebay-notice` Usage
+
+```marko
+<ebay-notice heading-level="3" status="priority" type="page" aria-level="Priority">
```

Comment:
Make sure to update the `aria-level` reference here

Reply:
yes will do it now